### ISSUE TITLE
License Update?

### ISSUE THREAD

**Issue by s-mccarthy** [snok/django-auth-adfs#300]
Hello,

i’ve been using this package for a while now and it’s been great, a massive time saver! 

Recently, policies were updated/implemented in our Nexus IQ/Sonatype and the package is being blocked for a license violation. I can see it’s only a standard 2 clause BSD license. When I reviewed the license template against the one this package uses, the only difference I can see is the addition of “All rights reserved.” On the second line. 

Any chance this line can be removed and an updated package pushed?

**Comment by JonasKs** [comment]
Yes, please create a PR if you can. If not I'll fix when I get home next week.

**Comment by s-mccarthy** [comment]
Thanks, will get to it later. Appreciate the quick response.

**Comment by stefan6419846** [comment]
> When I reviewed the license template against the one this package uses, the only difference I can see is the addition of “All rights reserved.” On the second line.

Is this really an issue for license compliance? According to section B.11.2 of the SPDX 2.3 specification, this is an ignorable part of the copyright: https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b35-guideline-omittable-text (There could be a debate whether the [template](https://github.com/spdx/license-list-XML/blob/main/src/BSD-2-Clause.xml) should include an optional reference to the clause in question.)

Additionally, the bullet points are allowed to differ for matching purposes, see section B.8: https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b8-bullets-and-numbering

Apart from this: With #307 having changed this to match the default template, I guess this issue can be closed?

**Comment by s-mccarthy** [comment]
Yes, it can be closed. ThanksOn 15 Jan 2026, at 14:04, Stefan ***@***.***> wrote:﻿stefan6419846 left a comment (snok/django-auth-adfs#300)

When I reviewed the license template against the one this package uses, the only difference I can see is the addition of “All rights reserved.” On the second line.

Is this really an issue for license compliance? According to section B.11.2 of the SPDX 2.3 specification, this is an ignorable part of the copyright: https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b35-guideline-omittable-text (There could be a debate whether the template should include an optional reference to the clause in question.)
Additionally, the bullet points are allowed to differ for matching purposes, see section B.8: https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b8-bullets-and-numbering
Apart from this: With #307 having changed this to match the default template, I guess this issue can be closed?
